pkg/v1/resilience/README.md

```
@@ -18,6 +18,20 @@ err := r.Run(ctx, func(ctx context.Context) error {
 
 Rejections surface typed sentinels \(RetryExhausted / CircuitOpen / RateLimited / BulkheadFull / TimeoutExceeded\) matchable via errs.HasReason / errs.HasCode.
 
+### Classifying deterministic failures
+
+Retry and the circuit\-breaker act on transient failures. A deterministic one — a policy refusal, an HTTP 400, invalid input — gains nothing from a replay and says nothing about a dependency's health, yet by default both policies treat every non\-nil error as transient: the retry spends its whole budget on it and hides it behind RetryExhausted, and the breaker counts it towards tripping. Set the Retryable predicate on either config to tell the two apart. It takes the same shape in both, so a nested Retry\(Breaker\(op\)\) shares one classifier. The sentinel is the caller's own — this package exports none to classify against, because only the caller knows which of their failures are deterministic:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)